### PR TITLE
relnote(Fx114): CSS calc() function supports NaN and infinity

### PR DESCRIPTION
**Pull request by bsmth**
Relnote & updates to docs due to handling of `infinity` and `NaN` inside the CSS `calc()` function.

__Related issues and pull requests:__
- [x] Parent issue https://github.com/mdn/content/issues/26693

__Bugzilla:__
- Tracking bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1830759

__TODO:__
- [x] Move constants info out to `calc-constants` page

__Other resources:__
- spec: https://drafts.csswg.org/css-values/#calc-error-constants
- [GDoc - Infinity and NaN for calc() (Author: Seokho Song)](https://docs.google.com/document/d/1kksm8aa5HpCph5NmJEwrCrj2e3p85RORQCr7OSsWAOs/edit#heading=h.7nki9mck5t64)
- codepen https://codepen.io/bsmth/pen/JjmBegM
- Implementation details in browser engines can be found in the linked BCD pull request

__related issues and pull requests:__
- [x] relevant BCD issue https://github.com/mdn/browser-compat-data/issues/19168
- [x] BCD https://github.com/mdn/browser-compat-data/pull/19882

### CODE REVIEW

File: files/en-us/web/css/calc-constant/index.md [hunk 1]
```diff
@@ -70,14 +88,137 @@ InFiNiTy
...
+```css
+div {
+  height: 50px;
+  background-color: red;
+  width: calc(1px / 0);
+}
+```
+
+```js
+const div = document.getElementById("div");
```
Comment (Rumyra) [inline files/en-us/web/css/calc-constant/index.md:200]:
```suggestion
const div = document.querySelector("div");
```

File: files/en-us/web/css/calc-constant/index.md [hunk 2]
```diff
@@ -70,14 +88,137 @@ InFiNiTy
...
+logSerializedWidth("calc(1px / 0)"); // calc(infinity * 1px)
+logSerializedWidth("calc(1px / -0)"); // calc(-infinity * 1px)
+
+logSerializedWidth("calc(1px * -infinity * -infinity)"); // calc(infinity * 1px)
+logSerializedWidth("calc(1px * -infinity * infinity)"); // calc(-infinity * 1px)
+
+logSerializedWidth("calc(1px * (NaN + 1))"); // calc(NaN * 1px)
+
+// reset
+div.style.width = "50px";
```
Comment (Rumyra) [inline files/en-us/web/css/calc-constant/index.md:217]:
```suggestion
```

**Review by Rumyra — CHANGES_REQUESTED** [review]

File: files/en-us/web/css/calc-constant/index.md [hunk 3]
```diff
@@ -70,14 +88,137 @@ InFiNiTy
...
+
+piInput.addEventListener("input", function () {
+  pi.style.rotate = "calc(sin(" + this.value + " * pi) * 100deg)";
+  piValue.textContent = pi.style.rotate;
+});
+```
+
+{{EmbedLiveSample('Using_e_and_pi_in_calc', 'auto', '200')}}
+
+### Infinity, NaN, and division by zero
```
[the remark this answers is not in the document]
Comment (bsmth) [reply files/en-us/web/css/calc-constant/index.md:183]: Good call 👍🏻

File: files/en-us/web/css/calc-constant/index.md [hunk 4]
```diff
@@ -70,14 +88,137 @@ InFiNiTy
...
+  eValue.textContent = e.style.transform;
+});
+
+piInput.addEventListener("input", function () {
+  pi.style.rotate = "calc(sin(" + this.value + " * pi) * 100deg)";
+  piValue.textContent = pi.style.rotate;
+});
+```
+
+{{EmbedLiveSample('Using_e_and_pi_in_calc', 'auto', '200')}}
```
[the remark this answers is not in the document]
Comment (bsmth) [reply files/en-us/web/css/calc-constant/index.md:175]: Could you have another look at https://pr26784.content.dev.mdn.mozit.cloud/en-US/docs/Web/CSS/calc-constant#using_e_and_pi_in_calc ? It should be working ™️

**Review by Rumyra — APPROVED** [review]

**Pull request merged**